spec/0.0.2/README.md

```diff
@@ -262,33 +262,41 @@ Field Name | Type | Description
 
 #### <a name="artifactsObject"></a>Artifacts Object
 
-The Artifacts Object describes a list of provider specific artifact items. These artifact items will be used during installation of the containerapp to deploy it to the provider. Each artifact item is a `URL`.
+The Artifacts Object describes a list of provider specific artifact items. These artifact items will be used during installation of the containerapp to deploy it to the provider. Each provider key contains a list of artifacts. Each artifact list item is either a `URL` string or a [source control repository object](#repositoryObject).
+
+* URL: must be a URL string prepended by type such as `http://`, `https://` or `file://`. URL type `file://` may be a single file or a directory path to multiple files. Directories must end with a trailing slash such as `file://relative/path/to/multiple/artifact/files/`.
```

Comment:
Ok, than maybe it's just a `urllib2` misbehaving - see http://pastebin.test.redhat.com/285486

* `file://` raises `URLError`
* `file:` works as expected
* `file:// + absolute path` works as expected (not attached in the pastebin)

But from what I understood from http://en.wikipedia.org/wiki/File_URI_scheme - once there are two slashes `//`, the path is expected to be absolute - where first part of the path is expected to be host. 

* `file://localhost/etc/hosts` - path to `/etc/hosts` on `localhost`
* `file:///etc/hosts` - same as above, `localhost` is automatically added
* `file://etc/hosts` - path to `/hosts` on the host `etc`

But we cannot specify absolute path in the nulecule. So I'd use what urllib provides with `file:` being a relative path as I expect it to be a de facto standard.